index.js

```diff
@@ -13,7 +13,6 @@ const jsonclone = require('./utils/jsonclone');
 
 const TRANSLATIONS = typeof Symbol === 'function' ? Symbol() : '_translations';
 const MISSING = typeof Symbol === 'function' ? Symbol() : '_missing';
-const MEMORY = typeof Symbol === 'function' ? Symbol() : '_memory';
```

Comment:
Might be a good opportunity to get rid of this check altogether.
`Symbol` is supported fully from `^6.5.0` - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol#Browser_compatibility.

Reply:
I don't mind supporting IE with this tiny check. It's not extremely important that it's a Symbol.